### DISCUSSION TITLE
SC 2.4.7 Does Focus Visible also apply in zoom state (200, 400 percent)?

### DISCUSSION THREAD

**Discussion by Ignacio-de-Loyola** [w3c/wcag#3411] [Q&A]
If the focus indicator does partially visible or does not appear on the screen when zoomed in 200(1.4.4), 400(1.4.10) percent zoom state, is this considered a SC failure?

**Comment by aparnapasi** [comment]
I would say yes, to the best of my knowledge, there are no such exceptions specified in 2.4.7

**Reply by Ignacio-de-Loyola** [reply]
Are there any conditions regarding what resolution it is?

**Reply by patrickhlauke** [reply]
No, the requirements are required at all resolutions unless an limit to their applicability is explicitly mentioned in the SC

**Reply by patrickhlauke** [reply]
however, note that 2.4.7 doesn't require that the focus indicator be fully visible. if it's only partially visible, it nominally passes 2.4.7 (but in the upcoming WCAG 2.2 the new 2.4.11 Focus Not Obscured (Minimum) and 2.4.12 Focus Not Obscured (Enhanced) deal with this aspect
